docs/BUILDING.md

```diff
@@ -275,6 +275,29 @@ export ANDROID_NDK_HOME=~/Library/Android/sdk/ndk/21.0.6113669
 make -f Makefile-Android
 ```
 
+### Generate the documentation
+
+The doxygen documentation is generated automatically if doxygen is detected on
+the system. The generated documentation can be found into
+`build/<CONFIG>/docs/html`
```

Comment:
The root of the path will depend on how and where the developer built. Probably best to describe this as "`${top_builddir}/docs/html` where `${top_builddir}` varies depending on how the project was built.".

Reply:
Done.